.github/labeler.json

```
@@ -7,7 +7,8 @@
       "packages/next/**",
       "packages/react-dev-overlay/**",
       "packages/react-refresh-utils/**",
-      "packages/next-codemod/**"
+      "packages/next-codemod/**",
+      "packages/eslint-plugin-next/**"
```

```suggestion
      "packages/eslint-plugin-next/**",
      "packages/eslint-config-next/**",
      "packages/next-env/**"
```
It might be good to add these two as well